### ISSUE TITLE
Make Stream recordable

### ISSUE THREAD

**Issue by kueken** [zgemma-star/servicehisilicon#15]
There is still no option to record a stream.
It is only returned with return.
It would be nice if this function could be implemented.

**Comment by Dima73** [comment]
Yes,example [IPTV recording in E2](https://github.com/OpenPLi/enigma2/commit/61a22ab277f3e228663daccb188dc4a6a164abc0)

And need add timeshift.
